.circleci/config.yml

```diff
@@ -84,52 +85,57 @@ defaults: &defaults
 
 version: 2
 jobs:
-  test-node-8:
+  test-javascript-node-8:
     <<: *defaults
     docker:
       - image: circleci/node:8
     steps:
       - checkout
-      - run: npm install --no-package-lock
+      - run: 
+          name: Install Dependencies
+          command: |
+            npm config set spin=false
+            npm config set progress=false
```

Comment:
Can we [make a common step](https://github.com/hramos/react-native/blob/4c1ef4ed4dc8e0b168d53321c67778271a6bfe36/.circleci/config.yml#L57-L61) for `npm install` just like we did for cache restoration and use it like [this](https://github.com/hramos/react-native/blob/4c1ef4ed4dc8e0b168d53321c67778271a6bfe36/.circleci/config.yml#L301)? We seem to be repeating that over and over in many places.

Reply:
Agreed. I like what you did in the other PR.